### ISSUE TITLE
[Issue]: HIPCC not executing, fails with path error

### ISSUE THREAD

**Issue by aaronsuydam** [ROCm/ROCm#3680]
### Problem Description

I am trying to compile some of the HIP samples on my computer, I downloaded and installed the HIP SDK and everything seemed fine, I had to install perl, as mentioned in #2713, but then I tried to compile the vector add samples, and I get this output: 
```
PS C:\Users\suyda\dev\school\research\GPU Frameworks Project\ROCm\HIP-Examples\vectorAdd> hipcc .\vectoradd_hip.cpp -o .\vectoradd_hip.exe
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.

failed to execute:C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.3\bin/nvcc  -Wno-deprecated-gpu-targets  -isystem C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.3/include -isystem "include" -x cu  -Wno-deprecated-gpu-targets -lcuda -lcudart -LC:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.3/lib64  .\vectoradd_hip.cpp -o ".\vectoradd_hip.exe"
PS C:\Users\suyda\dev\school\research\GPU Frameworks Project\ROCm\HIP-Examples\vectorAdd>
```

Not sure what to do. 

### Operating System

Windows 11

### CPU

AMD Ryzen 7 7600X

### GPU

AMD Instinct MI250

### ROCm Version

ROCm 6.1.0

### ROCm Component

HIPCC

### Additional Information

I don't actually have an MI250, it is making me pick a gpu anyway to report the issue.

**Comment by harkgill-amd** [comment]
Hi @aaronsuydam, this may be due to `hipcc` not being added to the systems PATH. Could you try calling the executable directly from `C:\Program Files\AMD\ROCm\6.1\bin`? If this works, adding this directory to the systems PATH through the Environment Variables menu in Windows should allow you to call `hipcc` regardless of which directory you are in.

**Comment by harkgill-amd** [comment]
@aaronsuydam, thanks for checking. 

There is currently a bug with how hipcc handles spaces in Windows paths. A fix will be available for this in the next Windows HIP SDK release though a temporary workaround may be to set the environment variable `HIP_PLATFORM=amd`. Could you please give this a try and let me know if the behavior differs. Please also note that the integrated graphics from the Ryzen 7600X are not currently supported for use with the Windows HIP SDK. This is not the cause of this specific issue but may result in different errors arising.

**Comment by aaronsuydam** [comment]
@harkgill-amd makes sense, another workaround that I found is running the bin/hipcc.pl perl script directly doesn't cause the error.

Theoretically if I wanted to enable my iGPU on my ryzen 7600x, is there a way to do that even though that means giving up the support and such? I know it isn't officially supported but I'm wondering if it's even possible.

**Comment by harkgill-amd** [comment]
@aaronsuydam, there is no specific workaround to enable the iGPU for the Windows HIP SDK though there may still be some usability maintained. I will provide an update when the fix is made available and you can give it a try then to confirm.

**Comment by harkgill-amd** [comment]
Hi @aaronsuydam, the 6.2.4 HIP SDK for Windows has been released which contains the fix for the handling of spaces in Windows paths. I can no longer reproduce the issues on my end. Could you please give this a try and confirm if the issue has been resolved?

**Comment by harkgill-amd** [comment]
Closing this issue out due to a lack of response. If you're still experiencing the path errors on the latest HIP SDK, please leave a comment and I'll re-open this issue. Thanks!
